src/System.Security.Cryptography.Cng/ref/Configurations.props

```diff
@@ -1,14 +1,15 @@
 ﻿<Project DefaultTargets="Build">
   <PropertyGroup>
     <PackageConfigurations>
-      netcoreapp;
+      netcoreapp3.0;
```

Comment:
I don't understand the implications of this.  Presumably if we make changes in master at this point we want them to be in the package, so I'd expect netcoreapp to still be here.

This library is actually inbox on Windows' NETCore.App; but it builds as OOB to keep it out of the default reference set.  I don't know if that impacts what the right answer here is.

Reply:
Most of the crypto packages are in "dual" mode: both inbox and out-of-box, like Immutable, SRM and others.  As a result whenever we bump the TFM we need to make sure we don't drop the asset from last release from the package.  I tagged you here so you can review what's happening across all the crypto packages to make sure it makes sense to you.  This should illustrate the cost/side-effects of letting the crypto libraries cross-compile against the latest netcoreapp.

Reply:
Right, the problem is I'm basically reviewing without understanding :smile:.

I _think_ that what this says is the netcoreapp3.0 ref-asset needs to be in the package (presumably it gets harvested), this seems fine.

I also think it means that the netcoreapp-current (netcoreapp5.0 now) one is not in the package, but we're still compiling it... and that seems bad (because any changes effectively go nowhere).

If we're trying to prevent TFM creep and we also limited the src/ project to build as netcoreapp3.0 (instead of netcoreapp-current) then that would be OK, I suppose... right up until it isn't.  Though I note the netcoreapp2.1 contents aren't here... so I don't know if that means the package is in an invalid state, or if that line is being fed by netstandard-something, or what.  (Fake OOB confuses me)

Reply:
Not harvested: built.  Right now we will build 3.0 and 5.0 as identical and only pack 3.0.  We need to build a 5.0 for source build.  We avoid package growth by keeping it the same as 3.0